### ISSUE TITLE
Vertical-synchronization related issues

### ISSUE THREAD

**Issue by EmptyDreams** [jMonkeyEngine/jmonkeyengine#1836]
When I turned on vsync, the frame rate was limited to 30FPS, but my monitor actually refreshed at 60Hz. When I turned off vsync, the frame rate stabilized at 60FPS.
I want to know what caused this.

**Comment by tonihele** [comment]
What version of jME are you using? Sounds like this one issue that has been fixed already awhile ago.

**Comment by EmptyDreams** [comment]
> What version of jME are you using? Sounds like this one issue that has been fixed already awhile ago.

```
def jme3 = [v:'3.4.0-stable', g:'org.jmonkeyengine']
"${jme3.g}:jme3-core:${jme3.v}"
"${jme3.g}:jme3-desktop:${jme3.v}"
"${jme3.g}:jme3-lwjgl3:${jme3.v}"
"${jme3.g}:jme3-plugins:${jme3.v}"
"${jme3.g}:jme3-jogg:${jme3.v}"
```

**Comment by tonihele** [comment]
Yep, the bug was in LWJGL3 version of jME. You seem to be using LWJGL 3. Can you try jME version 3.5.2?

**Comment by EmptyDreams** [comment]
> Yep, the bug was in LWJGL3 version of jME. You seem to be using LWJGL 3. Can you try jME version 3.5.2?

Yes, this issue was successfully resolved after I updated 3.5.2
